cli-plugins/hooks/printer.go

```diff
@@ -0,0 +1,18 @@
+package hooks
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/morikuni/aec"
+)
+
+func PrintNextSteps(out io.Writer, messages []string) {
+	if len(messages) == 0 {
+		return
+	}
+	fmt.Fprintln(out, aec.LightBlueF.Apply("\nNext steps:"))
+	for _, n := range messages {
+		fmt.Fprintf(out, "    - %s\n", n)
```

Comment:
nit: as we're creating a wrapper here, should we appease the linters, and `_, _ = ` for these?